gateway/coprocess_bundle_test.go

```diff
@@ -422,7 +375,8 @@ func TestResponseOverride(t *testing.T) {
 	})
 }
 
-func TestPullBundle(t *testing.T) {
+func TestBundle_Pull(t *testing.T) {
+	t.Skip()
```

Comment:
?

Reply:
Renamed to follow typical go test conventions, Test<Struct>_<Func/Comment>, TestBundle_Verify below.